README.md

```diff
@@ -323,6 +323,10 @@ as `onNext`, `onComplete`, or `cancel`.
 
 `take%%<n>%%<id>` : The driver should hang the test thread until it has received at least `n` items IN TOTAL, and then immediately cancel.
 
+`disconnect` : The driver should disconnect the client from the server.
+
+`resume` : The driver should resume the connection with the server.
```

Comment:
To generate any error, server can send "#" . Do we need any additional support ? Do we need to care about the type of error at TCK level? 
I will add test cases with errors